### PR TITLE
KBHDP SOA Case Study

### PR DESCRIPTION
**Pull request by zacharybinger**
Creating PR for SOA Case Study

### CODE REVIEW

File: src/watertap_contrib/reflo/analysis/case_studies/KBHDP/KBHDP_SOA.py
Comment (adam-a-a) [inline src/watertap_contrib/reflo/analysis/case_studies/KBHDP/KBHDP_SOA.py:69]: delete?

File: src/watertap_contrib/reflo/analysis/case_studies/KBHDP/KBHDP_SOA.py
Comment (adam-a-a) [inline src/watertap_contrib/reflo/analysis/case_studies/KBHDP/KBHDP_SOA.py:210]: delete?

File: src/watertap_contrib/reflo/analysis/case_studies/KBHDP/components/__init__.py
Comment (adam-a-a) [inline src/watertap_contrib/reflo/analysis/case_studies/KBHDP/components/__init__.py:4]: Typically I would think we'd be careful with wildcard imports.

**Review by adam-a-a — APPROVED** [review]
LGTM - just some cleanup if you want to polish before merge

File: src/watertap_contrib/reflo/analysis/case_studies/KBHDP/KBHDP_SOA.py
Comment (zacharybinger) [reply src/watertap_contrib/reflo/analysis/case_studies/KBHDP/KBHDP_SOA.py:210]: Yup!

**Review by kurbansitterley — APPROVED** [review]
LGTM - tests and clean up/finalization once we have final results.

**Pull request merged**
